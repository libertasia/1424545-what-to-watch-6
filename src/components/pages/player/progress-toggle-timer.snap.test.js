import React from 'react';
import {Provider} from 'react-redux';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import configureStore from 'redux-mock-store';
import {createMemoryHistory} from 'history';
import ProgressTogglerTimer from './progress-toggle-timer';

const mockStore = configureStore({});

it(`ProgressTogglerTimer renders correctly`, () => {
  const history = createMemoryHistory();
  const progress = 0;
  const timer = `05:05`;

  const {container} = render(
      <Provider store={mockStore({})}>
        <Router history={history}>
          <ProgressTogglerTimer
            progress={progress}
            timer={timer}
            onProgressClickHandler={jest.fn()} />
        </Router>
      </Provider>
  );

  expect(container).toMatchSnapshot();
});
