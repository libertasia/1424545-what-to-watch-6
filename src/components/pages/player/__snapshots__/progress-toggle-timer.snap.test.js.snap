// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressTogglerTimer renders correctly 1`] = `
<div>
  <div
    class="player__controls-row"
  >
    <div
      class="player__time"
    >
      <progress
        class="player__progress"
        max="100"
        value="0"
      />
      <div
        class="player__toggler"
        style="left: 0%;"
      >
        Toggler
      </div>
    </div>
    <div
      class="player__time-value"
    >
      05:05
    </div>
  </div>
</div>
`;
